packages/zoe/zoe.js

```diff
@@ -9,31 +9,28 @@ import { isOfferSafeForAll } from './isOfferSafe';
 import { areRightsConserved } from './areRightsConserved';
 import { evalContractCode } from './evalContractCode';
 import { makeTables } from './state';
-import { makeSeatMint } from './seatMint';
-import { makeEscrowReceiptConfig } from './escrowReceiptConfig';
+import { makeInviteConfig } from './inviteConfig';
 
 /**
  * Create an instance of Zoe.
  *
  * @param additionalEndowments pure or pure-ish endowments to add to evaluator
  */
 const makeZoe = (additionalEndowments = {}) => {
-  // Zoe has two mints: a mint for invites and a mint for
-  // escrowReceipts. The invite mint can be used by a smart contract
-  // to create invites to take certain actions in the smart contract.
-  // An escrowReceipt is an ERTP payment that is proof of
-  // escrowing assets with Zoe.
-  const {
-    seatMint: inviteMint,
-    seatAssay: inviteAssay,
-    addUseObj: addInviteSeat,
-  } = makeSeatMint('zoeInvite');
-
-  const escrowReceiptMint = makeMint(
-    'zoeEscrowReceipts',
-    makeEscrowReceiptConfig,
-  );
-  const escrowReceiptAssay = escrowReceiptMint.getAssay();
+  const makeInviteMint = () => {
+    const handleToSeat = new WeakMap();
+    const inviteMint = makeMint('zoeInvite', makeInviteConfig);
+    return harden({
```

Comment:
I was confused by this about three times reading the code. The function on line 20 is called `makeInviteMint`. The variable on line 22 is called `inviteMint`. But the object named by that variable is not what a call to `makeInviteMint` returns. The object that is returned does not have the API described by mint.chainmail, and so is not even a Mint.

Reply:
Ok, it seems like the name of the function is what is creating the wrong expectation. I'll change the name of the function. I think other than that, it's pretty straightforward. I'm encapsulating all of the invite logic so all that Zoe needs to know is that there is an `inviteAssay`, and two functions, `mintInvite` and `getInviteSeat`. I think the encapsulation is valuable. Zoe shouldn't have to know about `handleToSeat`.

Reply:
I said this in a different comment thread, but I ended up taking this code out of the function. Let me know if that works better for you!

Reply:
I like the way it reads now. Addresses all my concerns here. Resolving.